x-pack/plugins/security_solution/server/endpoint/services/actions/utils.test.ts

```diff
@@ -147,6 +159,42 @@ describe('When using Actions service utilities', () => {
         completedAt: COMPLETED_AT,
         errors: undefined,
         wasSuccessful: true,
+        ...EXPECTED_OUTPUTS,
+      });
+    });
+
+    it('should show running processes in outputs', () => {
```

Comment:
Suggestion: rename to `should return action outputs (if any) per agent id`
Although the example below is using running processes, the intent of the utility to to just grab the output for an agent id regardless of what action it actually is.